vars/unitTestPost.groovy

```diff
@@ -45,45 +52,51 @@ def call(Map config = [:]) {
     return
   }
 
-  junit testResults: config.get('testResults', 'test_results/*.xml')
+  def test_results = config.get('testResults', 'test_results/*.xml')
+  if (test_results != "") {
+    junit testResults: test_results
+  }
 
   def artifact_list = config.get('artifacts', ['run_test.sh/*', 'vm_test/**'])
   artifact_list.each {
     archiveArtifacts artifacts: it
   }
-
+  
+  def valgrind_pattern = config.get('valgrind_pattern', 'dnt.*.memcheck.xml')
```

Comment:
We are doing already doing some valgrind tests?

Reply:
yes, it's enabled for run_test.sh, I'm working on your comments, thanks